libs/sdk-ui-kit/src/Dialog/ShareDialog/ShareDialog.tsx

```diff
@@ -61,6 +84,10 @@ export const ShareDialog: React.FC<IShareDialogProps> = (props) => {
                         shareStatus={shareStatus}
                         sharedObjectRef={sharedObjectRef}
                         owner={owner}
+                        isLocked={isLocked}
+                        isUnderStrictControl={isUnderStrictControl}
+                        isLockingSupported={isLockingSupported}
+                        labels={labels}
```

Comment:
Labels needs to go throw all dialog comp tree maybe it will be fine to have provider for them and save some props in child components. In my impl. this is missing and there will be more components need it too.

Reply:
I will leave it as is. It will be fixed in the next PR that will add more labels.